wazo_acceptance/steps/device.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from behave import then, when
+
+
+@then('there are no devices with mac "{mac}"')
+def then_there_are_no_devices_with_mac(context, mac):
+    context.helpers.provd.delete_device_with_mac(mac)
```

Comment:
The step is indeed a cleanup step. It should be more explicit. The test creates a device by simulating a phone request made to provd through HTTP. I do not think it would be appropriate to add a cleanup to the HTTP request step since it is too generic. I think it is the only place that it is used, so I could create a step like "When creating a device with MAC xx:xx:xx:xx:xx:xx via a phone request". It would be then possible to cleanup properly the device.

Reply:
Yeah good idea.
If there are used in other place, I don't have any objection to have an explicit cleanup step (but more explicit than the actual)